versions.cfg

```diff
@@ -10,7 +10,9 @@ z3c.recipe.scripts = 1.0.1
 plone.recipe.zope2instance  = 4.4.0
 
 # Euphorie dependencies
+alembic = 1.1.0
```

Comment:
Do we need to add 
`python-editor = 1.0.4`
as well? It is a dependency of alembic.

Reply:
I missed it, thanks for noting it, I added this in d978ce2